## Slingshot Common Messages
## RA-11: Originally copied from Share, and everything except the
## required role.* and hybridsync.* properties removed.

## Roles
role.SiteManager=Gestionnaire
role.SiteCollaborator=Collaborateur
role.SiteContributor=Contributeur
role.SiteConsumer=Lecteur

#Hybrid Sync Console
hybridsync.ssd-id-header-label=ID de synchronisation
hybridsync.ssd-failed-reason-header-label=Type d'erreur de synchronisation
hybridsync.path-desc-header-label=Emplacement d'origine de l'\u00e9l\u00e9ment
hybridsync.sync-creator-header-label=Cr\u00e9ateur de synchronisation
hybridsync.cloud-user-header-label=Utilisateur cloud
hybridsync.remote-tenant-id-header-label=R\u00e9seau cloud
hybridsync.target-folder-node-ref-header-label=Destination cloud
hybridsync.actions-header-label=Actions

sync.folder.name_clash=Un contenu avec le m\u00eame nom existe d\u00e9j\u00e0 dans le dossier cible.
sync.folder.not_found=Aucun dossier n'a \u00e9t\u00e9 trouv\u00e9.
sync.node.already_synced=Impossible de cr\u00e9er le contenu\u00a0: il a d\u00e9j\u00e0 \u00e9t\u00e9 synchronis\u00e9 depuis un autre emplacement.
sync.node.other_sync_set=Ce contenu a d\u00e9j\u00e0 \u00e9t\u00e9 synchronis\u00e9 depuis un autre emplacement.
sync.node.no_longer_exists=Ce contenu n'existe plus dans Alfresco Content Services in the Cloud.
sync.node.no_longer_exists.description=Ce contenu n'existe plus dans Alfresco Content Services in the Cloud. Cliquez sur Demander la synchronisation pour r\u00e9essayer.
sync.node.unknown=Un probl\u00e8me est survenu, veuillez r\u00e9essayer.
sync.node.unknown.description=Une erreur inattendue s''est produite. Cause\u00a0: {0}.
sync.node.access_denied=Impossible de synchroniser le contenu\u00a0: vous ne disposez pas des permissions pour acc\u00e9der au fichier cible.
sync.node.access_denied_source=Impossible de synchroniser le contenu\u00a0: vous ne disposez pas des permissions pour acc\u00e9der au fichier source.
sync.node.content_limit_violation=Ce fichier d\u00e9passe le nombre maximum de contenus autoris\u00e9s. Le fichier est trop volumineux pour effectuer l'action.
sync.node.quota_limit_violation=Vous avez d\u00e9pass\u00e9 le quota autoris\u00e9 pour le contenu. L'espace libre est insuffisant pour effectuer l'action.
sync.node.authentication_error=Impossible de synchroniser le contenu sur le cloud\u00a0: les informations d'authentification ne sont pas valides.
sync.node.authentication_error.description=Impossible de synchroniser le contenu sur le cloud\u00a0: les informations d''authentification pour l''utilisateur {0} ne sont pas valides.
sync.node.owner_not_found=Impossible de synchroniser le contenu car l'utilisateur a \u00e9t\u00e9 supprim\u00e9.
sync.node.owner_not_found.description=L''utilisateur propri\u00e9taire de cette synchronisation, {0}, n''existe plus. Cliquez sur Se d\u00e9synchroniser.

sync.communications.error=Impossible de communiquer avec le syst\u00e8me distant
sync.communications.error.description=Impossible de communiquer avec le syst\u00e8me distant. La synchronisation fonctionnera automatiquement une fois les communications \u00e9tablies
